README.md

```diff
@@ -1 +1,232 @@
 # Project-UNICEF-COVID19-SOM
+Data pipeline for Project-UNICEF-COVID19-SOM.
+
+This pipeline fetches all project data from a Rapid Pro instance and Coda, and processes it to produce data for
+manual labelling and verification in Coda, CSV files suitable for analysis, and some automatic analysis of that data.
+
+## Pre-requisites
+Before the pipeline can be run, the following tools must be installed:
+ - Docker
+ - Bash
+ 
+Development requires the following additional tools:
+ - Python 3.6
+ - pipenv
+ - git
```

Comment:
Add a section that explains that you now need to pipenv --three sync after cloning

Reply:
updated thanks.

Reply:
@as2388  Please take another quick look.